doc/contribute/documentation/guidelines.rst

```diff
@@ -504,13 +570,128 @@ Would display as:
 There's a shorthand for writing code blocks too: end the introductory paragraph with a double colon
 (``::``) and indent the code block content that follows it by three spaces.  On output, only one
 colon will be shown.  The code block will have no highlighting (i.e. ``none``). You may however use
-the ``.. highlight::`` directive to customize the default language used in your document (see for
+the :rst:dir:`highlight` directive to customize the default language used in your document (see for
 example how this is done at the beginning of this very document).
 
+
+Links and Cross-References
+**************************
+
+.. _internal-linking:
+
+Cross-referencing internal content
+==================================
+
+Traditional ReST links are only supported within the current file using the
+notation::
+
+   Refer to the `internal-linking`_ page
+
+which renders as,
+
+   Refer to the `internal-linking`_ page
+
+Note the use of a trailing
+underscore to indicate an outbound link. In this example, the label was
+added immediately before a heading, so the text that's displayed is the
+heading text itself. You can change the text that's displayed as the
+link writing this as::
```

Comment:
```suggestion
link writing this as:
```

Does `::` have some special meaning?

Reply:
I think it's explained in the guidelines :)